[CDAP-13657] Fixes broken 'Engine config' tab in configure modeless for realtime pipelines

tonybach opened the pull request:
JIRA: https://issues.cask.co/browse/CDAP-13657

This regression in styling is due to the missing classes `configuration-content-batch` and `configuration-content-realtime`, which applies slightly different stylings for the pipeline config modeless, depending on whether they are batch or realtime pipelines. Also, currently we are showing `Pipeline alert` for realtime pipelines as well, which we shouldn't.

We used to have these classes in `ConfigurationsSidePanel` and `ConfigurationsContent` components, but we forgot to add them back in after we changed the way we render these pipeline configs (https://github.com/caskdata/cdap/commit/d555a859cc9845b9f23968456127be8ee7cf8e11#diff-056c60473baee727eeba8ea13631ce3e, https://github.com/caskdata/cdap/commit/d555a859cc9845b9f23968456127be8ee7cf8e11#diff-82de1d172b481eeaebdb08ce0c2836f3).

tonybach committed:
[CDAP-13657] Fixes broken 'Engine config' tab in configure modeless for realtime pipelines

ajainarayanan reviewed (commented):
Talked to @tonybach . Noticed one scroll issues with custom engine config. Otherwise 👍 Code LGTM

tonybach commented:
@ajainarayanan I noticed the 'Runtime arguments' tab in the Preview section was broken as well, since we removed its styling at `RuntimeArgsTabContent.scss`. Have added the file back in, and also addressed your previous comment.

ajainarayanan commented:
Talked to @tonybach and since he has other PRs he is working on I pushed the required change.

tonybach commented on cdap-ui/app/directives/my-pipeline-runtime-args/my-pipeline-runtime-args.less:
```diff
@@ -38,4 +38,13 @@ my-pipeline-runtime-args {
+    }
+  }
+  .runtime-arguments-values {
+    height: ~"calc(100% - 135px)";
```
nit: maybe make this `140px`. Right now we're showing a little bit of the 5th runtime arg, and it looks like extra border. 
![image](https://user-images.githubusercontent.com/6516002/42792804-eca71d50-892b-11e8-84d4-818b513f7723.png)

ajainarayanan replied:
So this is deliberate. If we make it `140px` user won't know if there are more run time arguments below. Have modified the margin of each key-value pair row so that we surface more of the bottom element so that it doesn't look like a border.

tonybach commented on cdap-ui/app/cdap/components/PipelineConfigurations/ConfigurationsContent/RuntimeArgsTabContent/RuntimeArgsTabContent.scss:
```diff
@@ -1,81 +0,0 @@
-      width: initial;
-    }
-
-    .provided-popover {
```
I think we still need some styling for the provided popover, otherwise it looks broken when you're using macros in Preview

ajainarayanan replied:
No. Have added styles that are needed in angular side.

elfenheart commented on cdap-ui/app/cdap/components/PipelineConfigurations/ConfigurationsContent/EngineConfigTabContent/EngineConfigTabContent.scss:
```diff
@@ -58,8 +58,9 @@ $input-label-color: $grey-03;
 
       .custom-config-values {
         /* 144px = 32px (height of input box) * 3
```
need to modify this 144 to 139

elfenheart reviewed (commented):
minor comment, otherwise 👍  LGTM

elfenheart reviewed (approved):
👍  code LGTM
